editor.planx.uk/src/@planx/components/Send/bops/index.ts

```diff
@@ -215,6 +227,7 @@ export const makePayload = (
         ];
       }
       metadata = addPortalName(id, flow, metadata);
+      metadata = addSectionName(id, metadata);
```

Comment:
This pattern felt a bit cleaner if `addPortalName()` is not going to be around for much longer.

Reply:
Is portal name used by BOPS currently?